packages/config/package.json

```diff
@@ -9,9 +9,10 @@
   },
   "license": "MIT",
   "type": "module",
-  "main": "dist/index.js",
```

Comment:
does the removal of `main` in these packages mean we can import plainly as TS? or does importing end up using the JS build?

Reply:
in vscode it's gonna resolve types immediately (via the `types` key), but during build it's gonna resolve to `dist/index.js` (via the `exports` key). This means to run tests etc we need to build first (which should be less of a pain once nx is added)